recipes/cabextract/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+./configure
+make
```

Comment:
Is there a make check we can run?

Reply:
At least I am not aware of such an option.